### ISSUE TITLE
Support VmWare vcloud

### ISSUE THREAD

**Issue by patsevanton** [pulumi/pulumi-vsphere#166]
Hello! Thanks for pulumi-vsphere provider.
Please create new provider VmWare vcloud or add VmWare vcloud to exist provider.
Thanks!

**Comment by masikrus** [comment]
some one do this?

**Comment by iwahbe** [comment]
Hi @patsevanton, can you be more specific in what you're looking for?

Has the functionality you needed been added to pulumi-vsphere? Is there an existing TF provider that does what you want?[^1]

[^1]: https://www.pulumi.com/registry/packages/terraform-provider/ would help you get started there.

**Comment by masikrus** [comment]
@iwahbe yap https://github.com/vmware/terraform-provider-vcd this one

**Comment by guineveresaenger** [comment]
Hi @masikrus - this provider is based on the [HashiCorp Terraform Provider for vsphere](https://github.com/hashicorp/terraform-provider-vsphere) which currently does not support this feature.

However, you should be able to create an as-needed local provider using https://github.com/vmware/terraform-provider-vcd and our [Any Terraform Provider](https://www.pulumi.com/registry/packages/terraform-provider/) as @iwahbe suggested.

We also encourage you to look into https://github.com/pulumi/pulumi-tf-provider-boilerplate if you would like to create a full bridged Pulumi provider against https://github.com/vmware/terraform-provider-vcd and are happy to help you with the publishing process.

**Comment by masikrus** [comment]
@guineveresaenger thx i got that, after 1-2 month i finish my bridge and publish )

**Comment by asilenkov** [comment]
> [@guineveresaenger](https://github.com/guineveresaenger) thx i got that, after 1-2 month i finish my bridge and publish )

Hello! Any progress you could share? )

**Comment by masikrus** [comment]
@asilenkov Yes. i got all ready worked provider for vcloud but not yet publish

**Comment by masikrus** [comment]
https://github.com/ergSey/pulumi-vcd - pulumi provider for Vcloud
